src/test/javascript/spec/component/tutorial-groups/registered-students/registered-students.component.spec.ts

```diff
@@ -0,0 +1,130 @@
+    importFinish: EventEmitter<void> = new EventEmitter();
+}
+
+describe('Registered Students Component', () => {
+    let comp: RegisteredStudentsComponent;
+    let fixture: ComponentFixture<RegisteredStudentsComponent>;
+    let tutorialGroup: TutorialGroup;
+    let tutorialGroupService: TutorialGroupsService;
+    let getTutorialGroupSpy: jest.SpyInstance;
+    const course = { id: 123, title: 'Example', isAtLeastInstructor: true };
+    const tutorialGroupUserOne = new User(1, 'user1');
+    const tutorialGroupUserTwo = new User(2, 'user2');
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [ArtemisTestModule],
+            declarations: [RegisteredStudentsComponent, CourseGroupStubComponent, MockDirective(TranslateDirective)],
+            providers: [
+                MockProvider(TutorialGroupsService),
+                {
+                    provide: ActivatedRoute,
+                    useValue: {
+                        paramMap: of({
+                            get: (key: string) => {
+                                switch (key) {
+                                    case 'tutorialGroupId':
+                                        return 1;
+                                }
+                            },
+                        }),
+                        parent: {
+                            parent: {
+                                data: of({ course }),
+                            },
+                        },
+                    },
+                },
+            ],
+        })
+            .compileComponents()
+            .then(() => {
+                fixture = TestBed.createComponent(RegisteredStudentsComponent);
+                comp = fixture.componentInstance;
+                tutorialGroupService = TestBed.inject(TutorialGroupsService);
+                tutorialGroup = new TutorialGroup();
+                tutorialGroup.title = 'Group';
+                tutorialGroup.id = 123;
+                tutorialGroup.course = course;
+                tutorialGroup.registeredStudents = [tutorialGroupUserOne, tutorialGroupUserTwo];
+                getTutorialGroupSpy = jest.spyOn(tutorialGroupService, 'getOneOfCourse').mockReturnValue(of(new HttpResponse({ body: tutorialGroup })));
+            });
+    });
+
+    afterEach(() => {
+        jest.restoreAllMocks();
+    });
+
+    it('should initialize', () => {
+        fixture.detectChanges();
+        expect(CourseGroupMembershipComponent).not.toBeNull();
+    });
+
+    describe('OnInit', () => {
+        it('should load tutorial group', () => {
+            fixture.detectChanges();
+            expect(comp.course).toEqual(course);
+            expect(comp.tutorialGroup).toEqual(tutorialGroup);
+            expect(comp.courseGroup).toEqual(CourseGroup.STUDENTS);
+            expect(getTutorialGroupSpy).toHaveBeenCalledTimes(1);
+            expect(comp.registeredStudents).toEqual(tutorialGroup.registeredStudents);
```

Comment:
Missing `toHaveBeenCalledWith()`

Reply:
Done in 26527db8ac4e1b72878a831b3e1e749b67b59361